### PR TITLE
Change Heimdall to be its own micro service and added support to talk to Heimdall from WebServer

### PR DESCRIPTION
**Pull request by chang47**

### CODE REVIEW

File: WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public WebAtomizerService(String serviceUrl) {
 
     public String homepage() {
         return "Hello there. You're talking to " + restTemplate.getForObject(
-                serviceUrl + "/tst",
+                serviceUrl + "/find/000gUYq5",
```
Comment (senghuot) [inline WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java]: we should take that out

File: WebServer/src/main/java/senghout/github/com/webserver/WebServerApplication.java [hunk 2]
```diff
@@ -25,11 +26,17 @@ RestTemplate restTemplate() {
 
 	@Bean
 	public WebAtomizerService atomizerService() {
-		return new WebAtomizerService("Atomizer");
+		return new WebAtomizerService("atomizer");
```
Comment (senghuot) [inline WebServer/src/main/java/senghout/github/com/webserver/WebServerApplication.java:29]: should we make this "Atomizer"

File: WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public WebAtomizerService(String serviceUrl) {
 
     public String homepage() {
```
Comment (senghuot) [inline WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java]: change the method name pls

File: WebServer/src/main/java/senghout/github/com/webserver/WebServerApplication.java (hunk 2 above)
Comment (chang47) [reply WebServer/src/main/java/senghout/github/com/webserver/WebServerApplication.java:29]: No

File: WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java (hunk 3 above)
Comment (chang47) [reply WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java]: Done

File: WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java (hunk 1 above)
Comment (chang47) [reply WebServer/src/main/java/senghout/github/com/webserver/WebAtomizerService.java]: Done

**Pull request merged**